message SectionUplinkPayload {
  required uint32 command_code = 1;
  required uint32 start_index = 2;
  required uint32 end_index = 3;
  required bytes section = 4;
}